task/fossa-scanner/0.1/fossa-scanner.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fossa-scanner
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, fossa
+    tekton.dev/displayName: "fossa scanner"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    fossa-cli is a zero-configuration polyglot dependency analysis tool. You can point fossa CLI at any codebase or build,
+    and it will automatically detect dependencies being used by your project.
+
+    This task can be used to scan for dependency analysis, license scanning, vulnerability scanning,
+    attribution report generation, and more on the source code in standalone mode.
+  workspaces:
+    - name: source
+      description: A workspace that contains the source code to scan.
+  params:
+    - name: FOSSA_API_KEY
+      description: API key to be passed to fossa cli.
+      type: string
+    - name: FOSSACLI_IMAGE
+      default: docker.io/nistal97/fossa-cli@sha256:b3dd65d1297e9436b9e0f176c355ea1fa9c6295d09f79032c830b6e7e3376293
```

Comment:
hey @nistal97, how is this image different from the one provided by [fossa](https://hub.docker.com/r/fossa/fossa-cli)?

Reply:
hey @vinamra28 , https://hub.docker.com/r/fossa/fossa-cli is the development img, it would require additional steps to download, install & configure cli tool, which mentioned in the page. So the default img here is "from fossa base image, and add fossa cli binary", to provide a convienient way to run fossa immediately